Signal/src/call/CallService.swift

```diff
@@ -994,7 +996,6 @@ protocol CallServiceObserver: class {
         Logger.debug("\(TAG) in \(#function)")
 
         PeerConnectionClient.stopAudioSession()
-        peerConnectionClient?.setDelegate(delegate:nil)
```

Comment:
Clearing of the delegate now occurs inside the next statement: `peerConnectionClient?.terminate()`. Before PCC clears its delegate, it invokes a few of the delegate method to ensure that the views & `CallService` release any references they have to the audio and video tracks.